### DISCUSSION TITLE
Why el-form still show english validation error message, while locale set to zh-cn ?

### DISCUSSION THREAD

**Discussion by AlwaysNoobCoder** [element-plus/element-plus#15583] [Q&A] [answered]
version: element-plus 2.4.4
![图片](https://github.com/element-plus/element-plus/assets/156557770/419774cc-2c71-4b35-966c-e74ed3d5e6d3)

![图片](https://github.com/element-plus/element-plus/assets/156557770/eea3c63d-8012-4ac5-b891-dbf120bfeba1)

**Comment by AlwaysNoobCoder** [comment]
I looked at the locale source code, form validation error message not covered at all...

why ? any reasonable cause for this ?

**Comment by chenxch** [comment, marked as answer]
This part has no meaning for internationalization, and most users will use their own messages.
